test/unit/action/wallet.spec.js

```diff
@@ -303,7 +308,31 @@ describe('Action Wallet Unit Tests', () => {
   describe('initWallet()', () => {
     it('should init wallet', async () => {
       grpc.sendUnlockerCommand.withArgs('InitWallet').resolves();
+      RNFS.unlink.resolves();
       await wallet.initWallet({ walletPassword: 'baz', seedMnemonic: ['foo'] });
+      expect(
+        RNFS.unlink,
+        'was called with',
+        '/foo/bar/data/chain/bitcoin/mainnet/wallet.db'
+      );
+      expect(
+        RNFS.unlink,
+        'was called with',
+        '/foo/bar/data/chain/bitcoin/testnet/wallet.db'
+      );
+      expect(store.walletUnlocked, 'to be', true);
+      expect(grpc.sendUnlockerCommand, 'was called with', 'InitWallet', {
+        walletPassword: Buffer.from('baz', 'utf8'),
+        cipherSeedMnemonic: ['foo'],
+      });
+      expect(nav.goSeedSuccess, 'was called once');
+    });
+
+    it('should not delete wallet if RNFS not supported', async () => {
```

Comment:
Curious in what case `RNFS` wouldn't be supported? Does this mean if `wallet.db` doesn't exist?

Reply:
That's handled in https://github.com/lightninglabs/lightning-app/pull/1253/files#diff-fbb9a2d7e2e2b04543043634746adfe1R292